### PR TITLE
[3] feat(twap): twap orders cancellation

### LINKED ISSUE #2584: TWAP order cancellation
Reference: https://github.com/rndlabs/composable-cow/blob/main/src/ComposableCoW.sol#L98

### PR DESCRIPTION
**Pull request by shoom3301**
Fixes #2584 

Just added a call of [ComposableCoW.remove](https://github.com/rndlabs/composable-cow/blob/main/src/ComposableCoW.sol#L98) for `ComposableCoW` orders cancellation

## To Test
1. Regular order cancellation
2. ETH-flow cancellation
3. ComposableCoW order cancellation

### CODE REVIEW

File: src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx [hunk 1]
```diff
@@ -190,7 +190,7 @@ export function OrderRow({
         </TableRowCheckboxWrapper>
       )}
       {/* Order sell/buy tokens */}
-      <styledEl.CurrencyCell clickable onClick={onClick}>
+      <styledEl.CurrencyCell clickable title={order.id} onClick={onClick}>
```
Comment (shoom3301) [inline src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx:193]: Added a small indicator to distinguish twap orders from others in the orders table.
If the id in the title is short - it's a ComposableCoW order :) 

image

File: src/common/hooks/useCancelOrder/useGetOnChainCancellation.ts [hunk 2]
```diff
@@ -14,17 +15,24 @@ import {
...
 
   return useCallback(
     (order: Order) => {
+      const composableCowId = order.composableCowInfo?.id
+
       const isEthFlowOrder = getIsEthFlowOrder(order)
 
+      if (composableCowId) {
+        return cancelTwapOrder(composableCowId)
+      }
```
Comment (alfetopito) [inline src/common/hooks/useCancelOrder/useGetOnChainCancellation.ts:28]: You can move the condition above if it takes precedence over ethflow order cancellation:

```suggestion
      if (composableCowId) {
        return cancelTwapOrder(composableCowId)
      }
      
      const isEthFlowOrder = getIsEthFlowOrder(order)
```

File: src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx (hunk 1 above)
Comment (alfetopito) [reply src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx:193]: Is this just for debugging?

**Review by alfetopito — APPROVED** [review]
Tested regular (on and off chain) and ethflow cancellations ✅

File: src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx (hunk 1 above)
Comment (shoom3301) [reply src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx:193]: Yes. Changed it to `data-id`, so now the tooltip won't be shown

File: src/common/hooks/useCancelOrder/useGetOnChainCancellation.ts (hunk 2 above)
Comment (shoom3301) [reply src/common/hooks/useCancelOrder/useGetOnChainCancellation.ts:28]: Thanks! Fixed

**Pull request merged**